lib/Git/Hooks.pm

```diff
@@ -1339,33 +1356,73 @@ These three options are required if you enable Gerrit hooks. They are
 used to construct the C<Gerrit::REST> object that is used to interact
 with Gerrit.
 
+=head2 githooks.gerrit.labels-pass VOTES
+
+This option defines which votes should be cast in which
+L<labels|https://gerrit-review.googlesource.com/Documentation/config-labels.html>
+to B<approve> a review in the Gerrit change when all verification hooks
+pass.
+
+VOTES is a comma-separated list of LABEL and VOTE mappings, such as:
+
+  Code-Review+2,Verification+1
+
+Which means that the C<Code-Review> label should receive a +2 and the label
+C<Verification> should receive a +1.
+
+If not specified, the default VOTES is:
+
+  Code-Review+1
+
+=head2 githooks.gerrit.labels-fail VOTES
+
+This option defines which votes should be cast in which
+L<labels|https://gerrit-review.googlesource.com/Documentation/config-labels.html>
+to B<reject> a review in the Gerrit change when some verification hooks
+fail.
+
+VOTES has the same syntax as above.
```

Comment:
Demorei para perceber o que "above" significava. Talvez seja melhor trocar esta linha por:

VOTES is a comma-separated list of LABEL and VOTE mappings, such as:

  Code-Review-2,Verification-1

Which means that the C<Code-Review> label should receive a -2 and the label
C<Verification> should receive a -1.